apps/vscode-extension/project.json

```diff
@@ -25,7 +25,8 @@
             "input": ".",
             "output": "."
           }
-        ]
+        ],
+        "updateBuildableProjectDepsInPackageJson": false
```

Comment:
why is this necessary?

Reply:
`@nrwl/js:tsc` executor is failing with error `Cannot find dist/package.json` if I don't disable it, I suspect the package.json are not copied to the dist yet when running this